app/views/admin/invitations/index.html.erb

```diff
@@ -1,45 +1,56 @@
 <% if FeatureFlag.enabled?(:member_index_view) %>
-  <div class="crayons-card p-3 s:p-4 m:p-7">
-    <header class="flex flex-col">
+  <%= javascript_packs_with_chunks_tag "admin/users/memberIndex", defer: true %>
```

Comment:
Now that this is pack is being reused in the invitations view (to similarly control dropdown behaviour), I'm thinking we may want to rename it 🤔

Reply:
Yes rename for least surprise, but perhaps as a follow-on?